index.md

```diff
@@ -4,23 +4,38 @@ layout: default
 
 _“Best practice templates and writing instructions for documenting open source software.”_
 
+### What?
 
-### Download
+Every project deserves a minimum level of _Good Docs_ that help users understand and solve problems with your product. But it can be really hard to know where to start or what to write. 
 
-* The [0.1 alpha straw-man release of templates](https://github.com/thegooddocsproject/templates/releases/tag/v0.1) was released at the 2019 Write The Docs - Australia conference.
-* The latest development version of templates are available from the [github main branch](https://github.com/thegooddocsproject/templates).
+We are creating processes, templates, and guides covering the numerous aspects of writing good documentation. We describe our logic, referencing research where available.
 
 ### Why?
 
-Every project deserves a minimum level of _Good Docs_ that help users understand and solve problems with your product. But it can be really hard to know where to start or what to write. How do you structure docs? What does a docs website look like? How do I keep docs consistently formatted? The list of questions far outweigh the answers.
+We want to help open source projects be more effective and more impactful by helping them lift the quality of their documentation. 
+
+Our open-source Minimum Viable Docset (MVD) will help you create a baseline set of documentation using a suite of templates.
+
+### Who?
 
-That’s where _The Good Docs Project_ fits in. Our open-source Minimum Viable Docset (MVD) will help you create a baseline set of _Good Docs_ using a suite of templates.
+We are writing for people who have an open source project and want to improve their docs.
 
-Your _Good Docs_ can then be extended into _Better Docs_ with greater depth, quality and maturity.
+Our target readers are software developers and early-stage technical writers, who can read and write English, and are motivated to learn by example.
```

Comment:
```suggestion
```
You can drop this line if you accept line above.

Reply:
Done.